src/components/yemen/education/resources.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import IRC from './IRC.png'
+import Iran from './Iran.png'
+import Crisis from './Crisis.png'
+import { Grid } from 'semantic-ui-react';
```

Comment:
Organize you imports like this: https://github.com/amanmibra/actfor/blob/master/src/components/yemen/donate/index.jsx#L1